### DISCUSSION TITLE
Reporting NS where unclear if >X results in I or R

### DISCUSSION THREAD

**Discussion by antunderwood** [msberends/AMR#10] [Ideas]
An update on #6
In the case of ABA and CIP the breakpoints are 1 and 4
>2 should be reported as R since the next log2 increment from 2 is 4

>1 WHONET reports as NS since it could be I or R

Could the change be made that for `conserve_capped_values`
if the `X` in `>X` is above the lower breakpoint but the next log2 increment is below the upper breakpoint it reports `NS` else if the next log2 increment above `X` matches or exceeds the upper breakpoint it reports `R`.

I understand if you want to leave as is since this breaks the simple R, I , S tri state paradigm

_Originally posted by @aunderwo in https://github.com/msberends/AMR/issues/6#issuecomment-666326826_

**Comment by msberends** [comment]
Oof, that's hard. We had to choose a 'truth' to go by when it comes to interpretation to R/SI. We chose to follow EUCAST, which means that I is not NS to us, but rather susceptible (albeit 'Susceptible, increased exposure'): please see our website https://msberends.github.io/AMR/reference/as.rsi.html#interpretation-of-r-and-s-i. 

From our colleagues in Sweden we understood last year that the WHO (including WHONET, EARS-Net) were also to adopt this idea. A quick internet search brought me to the [AMR Reporting Protocol 2020 from the ECDC](https://www.ecdc.europa.eu/sites/default/files/documents/EARS-Net-reporting-protocol-2020.pdf), where I found on page 13:

![image](https://user-images.githubusercontent.com/31037261/88928571-4775ce00-d279-11ea-9a47-608a1cd6672d.png)

I think it's wise to leave it as is. Unfortunately, the tri state paradigm is more complicated than we can capture in one methodology. Hopefully, being transparant and clear about our choices helps our users to understand how and why our package works the way it works.

But I'll keep thinking about it. If it seems more appropriate in a later phase to change the behaviour of `as.rsi()` the way you suggest, we'll implement it of course immediately.

_Originally posted by @msberends in https://github.com/msberends/AMR/issues/6#issuecomment-666368001_
